recipes/fig2dev/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "fig2dev" %}
+{% set version = "3.2.6a" %}
+{% set sha256 = "5e61a3d9a4f83db4b3199ee82dd54bb65b544369f1e8e38a2606c44cf71667a7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.xz
+  url: https://sourceforge.net/projects/mcj/files/{{ name }}-{{ version }}.tar.xz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - toolchain
+    - libpng
+    - jpeg
```

Comment:
Please pin `jpeg` and `libpng` as defined [here](https://github.com/conda-forge/conda-forge.github.io/blob/master/scripts/pin_the_slow_way.py#L28-L71) for both build and run requirements. Those pinnings are `jpeg 9*` and `libpng >=1.6.28,<1.7`